src/main/java/edu/nextstep/racingcar/car/Cars.java

```diff
@@ -0,0 +1,37 @@
+package edu.nextstep.racingcar.car;
+
+import edu.nextstep.racingcar.strategy.RacingStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<RacingCar> cars;
+
+    public Cars(List<RacingCar> racingCarList) {
+        this.cars = racingCarList;
+    }
+
+    public static Cars createCarList(int carNum) {
+        List<RacingCar> racingCarList = new ArrayList<>();
+        for (int i = 0; i < carNum; i++) {
+            racingCarList.add(new RacingCar());
+        }
+        return new Cars(racingCarList);
+    }
```

Comment:
```suggestion
    private Cars(List<RacingCar> cars) {
        this.cars = cars;
    }
    public static Cars of(List<RacingCar> cars) {
        return new Cars(cars);
    }

    public static Cars of(int carNum) {
        List<RacingCar> cars = new ArrayList<>();
        for (int i = 0; i < carNum; i++) {
            cars.add(new RacingCar());
        }
        return Cars.of(cars);
    }
```

이렇게 정적 생성 팩토리 메소드 패턴을 이용해서 `다형성`이 보장되는 형태로 작성해주면 더 좋을 것 같아요!